Sources/Build/describe().swift

```diff
@@ -128,6 +128,11 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
     #if os(Linux)
          args += ["-fPIC"]
     #endif
+    #if os(OSX)
+        if let sysroot = Toolchain.sysroot {
```

Comment:
You are correct it exists on Linux as well.

Does Toolchain.sysroot have a reasonable value on Linux (i.e. slash (/) )